nodes/mtdevice.py

```diff
@@ -1318,14 +1316,14 @@ def usage():
         ./mtdevice.py -y 3,2,1,0,0,0,0,0 -y 9,0,1,0,0,0,10,0
 
 SetUTCTime settings:
-    The time fields are set as follow:
-        ns: nanoseconds of second, range [0,1000000000]
+    The time fields are set as follows:
         year: range [1999,2099]
         month: range [1,12]
-        day: day of the month, [1,31]
-        hour: hour of the day, [0,23]
-        min: minute of hour, [0,59]
-        sec: seconds of minute, minute [0,59]
+        day: range [1,31]
+        hour: range [0,23]
+        min: range [0,59]
+        sec: range [0,59]
+        ns: range [0,1000000000]
```

Comment:
Ah, no, it's just that for the seconds, the description had the word minute twice instead of once.